src/external_connections/next_gen_io.py

```diff
@@ -35,10 +35,9 @@ def read_catchment_lateral_flows(path):
           catchment_id_list.append(catchment_id)
           catchment_qlats = pd.read_csv(file_name, names=["", catchment_id])
```

Comment:
Let's read the csv a little better to avoid the extra processing.
```python
# Read the second column of a csv file and return a series. The index will be an autoincrementing range.
catchment_qlats = pd.read_csv(file_name, names=[catchment_id], usecols=[1], sqeeze=True)
ql.append(catchment_qlats)
```
This removes the need to do the .iloc selection.